src/daml/Daml/Finance/App/Quoting/Model.daml

```diff
@@ -0,0 +1,41 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.App.Quoting.Model where
+
+import Daml.Finance.Interface.Types.Common.Types (Id, InstrumentQuantity)
+
+data Side
```

Comment:
I think we have the same type defined in the trading service as well, can probably re-use.

Reply:
agree - i just wanted to avoid having a cross-service dependency just for that. probably some common namespace would make sense here